apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -160,33 +157,29 @@ export default connect(
           );
         } else if (this.props.isAttached) {
           try {
-            if (this.props.showReactInspector) {
-              // parentheses prevent the object from being interpreted as a block rather than as an object
-              let result = this.props.evalInCurrentScope(
-                input[0] === '{' && input[input.length - 1] === '}'
-                  ? `(${input})`
-                  : input
-              );
-              result = this.props.jsInterpreter.interpreter.marshalInterpreterToNative(
-                result
-              );
-              this.appendLog({
-                output: result,
-                undefinedInput: input === 'undefined' ? true : false
-              });
-            } else {
-              const result = this.props.evalInCurrentScope(input);
-              this.appendLog('< ' + String(result));
-            }
+            // if (this.props.showReactInspector) {
```

Comment:
Are these leftover comments?

Reply:
Woops! Yes, I'll go ahead and remove those.